gatsby/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,39 @@
+import { useLocation } from "@reach/router";
+import { Link } from "gatsby";
+import React, { type FC } from "react";
+
+import { StackedNav, StackedNavLink } from "@nice-digital/nds-stacked-nav";
+
+export interface MenuProps {
+	ariaLabel?: string;
```

Comment:
If we keep the same prop types as the main stacked nav element then it'll be easier to swap out for the NDS one directly if/when we move the logic into the NDS